src/components/organisms/TemplateManagerPane/TemplateManagerPane.tsx

```diff
@@ -77,6 +79,10 @@ const TemplatesManagerPane: React.FC<Props> = ({height}) => {
     () => checkForExtensionsUpdates({templateMap, pluginMap, templatePackMap}, dispatch),
     [templateMap, pluginMap, templatePackMap, dispatch]
   );
+  const openHelpUrl = () => {
+    const repositoryUrl = `https://github.com/kubeshop/monokle/blob/main/docs/templates.md`;
```

Comment:
please move this URL to the `src/constants/constants.ts` file as `TEMPLATES_HELP_URL`

Reply:
done.